City-of-Helsinki/helsinki-paatokset
# PP-271: Finish declarations of affiliation page

To test:

- Checkout branch. Run `drush cr; drush cim -y`
- Create a landing page / basic page with url `tietoa-paatoksenteosta`. Add it to menu.
- Create a landing page / basic page with url `sidonnaisuusilmoitukset`. Add it to menu as a child link for the previously created page. This should result in a page that when viewed, is located at [https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta/sidonnaisuusilmoitukset](https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta/sidonnaisuusilmoitukset).
- Go to [https://helsinki-paatokset.docker.so/fi/admin/structure/taxonomy/manage/pdf_categories/overview](https://helsinki-paatokset.docker.so/fi/admin/structure/taxonomy/manage/pdf_categories/overview). Add two items, for example `Pormestarikunnat` and `Luottamushenkilöt`
- Add some DOA media items: [https://helsinki-paatokset.docker.so/fi/media/add/declaration_of_affiliation](https://helsinki-paatokset.docker.so/fi/media/add/declaration_of_affiliation). Make sure to set the pdf_category value for them.
- Visit your [sidonnaisuusilmoitukset](https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta/sidonnaisuusilmoitukset) page. It should display a list of declarations of affiliation. The menu should work the same as other pages.
